datafusion/proto/README.md

```diff
@@ -17,121 +17,13 @@
   under the License.
 -->
 
-# Apache Arrow DataFusion Proto
+# `datafusion-proto`: Apache Arrow DataFusion Proto Serialization / Deserialization
```

Comment:
```suggestion
# `datafusion-proto`: Apache Arrow DataFusion Protobuf Serialization / Deserialization
```

Maybe? Or better to keep consistent with crate name to try keep somewhat agnostic of the underlying imlpementation...?

Reply:
I think ProtoBuf is a more consistent name. 👍